sorting/merge_sort.cpp

```diff
@@ -82,8 +82,8 @@ void mergeSort(int *arr, int l, int r) {
  * sorting
  */
 void show(int *arr, int size) {
-    for (int i = 0; i < size; i++) std::cout << arr[i] << " ";
-    std::cout << "\n";
+    for (int i = 0; i < size; i++) std::cout << arr[i] << " "<<"\n";
+    
```

Comment:
the previous version was fine, it is more convenient to print array as a space seprated integers in a single line, it looks more readable.